src/domain/thirdpartyRequests/transactions.ts

```diff
@@ -176,7 +176,63 @@ async function forwardTransactionRequestError(
   }
 }
 
+/**
+ * @function forwardTransactionRequestNotification
+ * @description Forwards a PATCH transactions requests to destination PISP for processing
+ * @param {string} path Callback endpoint path
+ * @param {HapiUtil.Dictionary<string>} headers Headers object of the request
+ * @param {RestMethodsEnum} method The http method PATCH
+ * @param {string} payload Body of the received kafka commit message.
+ * @throws {FSPIOPError} Will throw an error if no endpoint to forward the transactions requests is
+ * found, if there are network errors or if there is a bad response
+ * @returns {Promise<void>}
+ */
+async function forwardTransactionRequestNotification(
+  headers: Hapi.Util.Dictionary<string>,
+  transactionRequestId: string,
+  payload: string,
+  path: string,
+  method: RestMethodsEnum,
+  ): Promise<void> {
+
+  // todo: this is a temporary interim endpoint we are using until a PATCH TPR transaction endpoint is added.
+  //       i.e TP_CB_URL_TRANSACTION_REQUEST_PATCH
+  const endpointType = Enum.EndPoints.FspEndpointTypes.TP_CB_URL_TRANSACTION_REQUEST_POST
```

Comment:
Cool, yeah thanks for this. Can you please open a follow up ticket?

Reply:
done. https://app.zenhub.com/workspaces/mojaloop-project-59edee71d1407922110cf083/issues/mojaloop/mojaloop/428